api/tests/security/csrf/test_csrf.py

```diff
@@ -36,27 +37,7 @@ def test_crsf_extension_get_new_csrf(app):
     assert 'csrf_token' in resp.json
     assert len(csrf_cookies) > 0
     assert 'Secure; HttpOnly; Path=/; SameSite=Lax' in csrf_cookies[0]
-
-def test_crsf_extension_get_existing_csrf(app):
-    """
-    When an app is built
-        and CSRF ext is applied
-            it should expose a new endpoint /csrf
-            when a csrf cookie is already set in the incoming request
-              it should return the csrf_token in a json
-    """
-    CSRF(app)
-    test_client = app.test_client()
-    test_client.set_cookie(os.getenv('SITE_URL'), 'csrf', 'csrf-value', samesite='Lax', httponly=True, secure=True)
-    resp = test_client.get('/csrf')
-    csrf_cookies = list(cookie_value for cookie_header, cookie_value in resp.headers if
-                        'Set-Cookie' in cookie_header and CSRF_COOKIE_NAME in cookie_value)
-
-    assert resp.status_code != 405
-    assert 'csrf_token' in resp.json
-    assert resp.json.get('csrf_token') == 'csrf-value'
-    assert len(csrf_cookies) > 0
-    assert 'Secure; HttpOnly; Path=/; SameSite=Lax' in csrf_cookies[0]
+    assert 'Expires=' in csrf_cookies[0]
```

Comment:
issue: let's assert the cookie actually expires in 30 seconds (by the way, I believe you could set `max_age` instead of `expires` if that helps)

Reply:
coming up in next rev